homeassistant/components/obihai/sensor.py

```diff
@@ -162,3 +178,5 @@ def update(self) -> None:
 
         if self._service_name in call_direction:
             self._state = call_direction.get(self._service_name)
+
+        self._state = None
```

Comment:
I think this line shouldn't be here. Leftover?

Reply:
Thanks, I think you're right.